canonical-websites/snapcraft.io
# Optimise the blog page and generated CSS

## Done
Return the images to the blog page with optimisation. Optimised the CSS generation. Fix the husky command.

## QA
- Go to /blog
- See that the images are back
- Check that the page weight is a significantly slimmer
